### DISCUSSION TITLE
2 factor signup tries to access passwords and passcode?

### DISCUSSION THREAD

**Discussion by lilshebeary** [community/community#129903] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

2 factor signup tries to access passwords and passcode? So, when asked to setup 2 factor auth, you are asked to scan a QR code, this QR opens up my passwords in my iPhone settings, why? It is supposed to open in GitHub and give me a code. I worry also about logging key events with my passcode. Is this what was intended? Being routed to saved passwords on the iPhone?

**Comment by ebndev** [comment]
Hi @lilshebeary, thanks for being a part of the GitHub Community!

It sounds like you've set up a passkey, which is an authentication method that many platforms use. You can read more about [passkeys](https://docs.github.com/en/authentication/authenticating-with-a-passkey/about-passkeys) to answer some of your questions.

If you'd prefer to use another method for 2FA, you can view our [documentation](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication) on setting up a TOTP app.
